templates/puppetserver-statefulset-compilers.yaml

```diff
@@ -67,16 +67,14 @@ spec:
               chmod +x /etc/puppetlabs/puppet/r10k_hiera_entrypoint.sh /etc/puppetlabs/puppet/r10k_hiera_cronjob.sh;
               {{- end }}
               {{- if .Values.hiera.config }}
-              cp /etc/puppetlabs/puppet/configmap/hiera.yaml /etc/puppetlabs/puppet/hiera.yaml;
+              cp /etc/puppetlabs/puppet/configmap/hiera.ya(not .Values.hiera.eyaml.existingSecret)ml /etc/puppetlabs/puppet/hiera.yaml;
```

Comment:
Please fix the if statement here. It's on the wrong line.

Reply:
Fixed